src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -481,5 +482,18 @@ public static bool IsWorkerDynamicConcurrencyEnabled(this IEnvironment environme
             }
             return false;
         }
+
+        public static HashSet<string> GetLanguageWorkerListToStartInPlaceholder(this IEnvironment environment)
+        {
+            string placeholderList = environment.GetEnvironmentVariableOrDefault(RpcWorkerConstants.FunctionWorkerPlaceholderModeListSettingName, string.Empty);
```

Comment:
Question: where do these environment variables get populated from in Windows? For example, for Linux it's a matter of adding them in the docker image, but for Windows?

Reply:
On Windows - we want to restrict starting a single language worker process - as all the runtimes are available on the windows VMs.  This environment will not be set in Windows.